app/views/newflow/login_signup/student_signup_form.html.erb

```diff
@@ -155,7 +157,8 @@
                 <div class="content">
                     <%= f.submit I18n.t(:"login_signup_form.continue_button"),
                                         data: { disable_with: I18n.t(:"login_signup_form.continue_button") },
-                                        class: 'primary', id: 'signup_form_submit_button' %>
+                                        class: 'primary', id: 'signup_form_submit_button',
+                                        :data => {:ga_category => 'Account Creation', ga_action: 'Click', ga_label: '3A-Email'} %>
```

Comment:
Similar as the facebook and google buttons. This will only track **how many times** someone **tried** to sign up with email and password. I want to make sure, is that really what we want?

Reply:
The requirements state to have it on the Continue button.

Reply:
Ok, that makes sense because we have another way of tracking actually successful signups (the Security Logs) 👍.